### PR TITLE
chore(deps): update aepp-sdk and aeproject

### PR DESCRIPTION
**Pull request by marc0olo**

### CODE REVIEW

File: test/aensDelegation.js [hunk 1]
```diff
@@ -104,15 +108,15 @@ describe('AensDelegation', () => {
...
             getOwnerDryRun = await contract.methods.get_owner(testName);
-            assert.equal(getOwnerDryRun.decodedResult, newOwnerKeypair.publicKey);
+            assert.equal(getOwnerDryRun.decodedResult, newOwnerAddress);
         });
         it('should revoke a name', async () => {
             // you need a signature from the new owner now ;)
-            aensDelegationSignature = await aeSdk.createAensDelegationSignature({ contractId: contract.deployInfo.address, name: testName }, { onAccount: newOwnerKeypair });
-            await contract.methods.revoke(newOwnerKeypair.publicKey, testName, aensDelegationSignature, {onAccount: delegateKeypair});
+            aensDelegationSignature = await aeSdk.createAensDelegationSignature(contract.deployInfo.address, { name: testName, onAccount: newOwnerAccount });
+            await contract.methods.revoke(newOwnerAddress, testName, aensDelegationSignature, {onAccount: delegateAccount});
```
Comment (marc0olo) [inline test/aensDelegation.js:119]: @davidyuk can you check why this fails now? is this a bug in the sdk?

File: test/aensDelegation.js [hunk 2]
```diff
@@ -34,16 +38,16 @@ describe('AensDelegation', () => {
...
+            onAccount: mainAccount,
+            denomination: AE_AMOUNT_FORMATS.AE
         });
 
         // fund newOwner with 1 AE as there is currently an issue (shouldn't be required for delegated actions)
         // see https://github.com/aeternity/aeternity/issues/3674
-        await aeSdk.spend(1, newOwnerKeypair.publicKey, {
-            onAccount: mainKeypair,
-            denomination: AmountFormatter.AE_AMOUNT_FORMATS.AE
+        await aeSdk.spend(1, nameOwnerAddress, {
```
Comment (davidyuk) [inline test/aensDelegation.js:48]: this was a hard debugging 😅
```suggestion
        await aeSdk.spend(1, newOwnerAddress, {
```

File: test/aensDelegation.js [hunk 3]
```diff
@@ -104,15 +108,15 @@ describe('AensDelegation', () => {
...
         });
         it('should revoke a name', async () => {
             // you need a signature from the new owner now ;)
-            aensDelegationSignature = await aeSdk.createAensDelegationSignature({ contractId: contract.deployInfo.address, name: testName }, { onAccount: newOwnerKeypair });
-            await contract.methods.revoke(newOwnerKeypair.publicKey, testName, aensDelegationSignature, {onAccount: delegateKeypair});
+            aensDelegationSignature = await aeSdk.createAensDelegationSignature(contract.deployInfo.address, { name: testName, onAccount: newOwnerAccount });
+            await contract.methods.revoke(newOwnerAddress, testName, aensDelegationSignature, {onAccount: delegateAccount});
             try {
                 nameResult = await aeSdk.aensQuery(testName);
             } catch(e) {
```
Comment (davidyuk) [inline test/aensDelegation.js:122]:
```suggestion
            } catch(e) {
              assert.equal(e.details.reason, 'Name revoked');
```

**Review by davidyuk — APPROVED** [review]

File: test/aensDelegation.js (hunk 2 above)
Comment (marc0olo) [reply test/aensDelegation.js:48]: sorry @davidyuk! seems like I fucked it up. but strange error anyway

**Pull request merged**